en/orm/table-objects.rst

```diff
@@ -140,6 +141,13 @@ Event List
 * ``Model.afterDelete``
 * ``Model.afterDeleteCommit``
 
+initialize
+----------
+
+.. php:method:: initialize(Event $event, ArrayObject $data, ArrayObject $options)
```

Comment:
so @markstory do you think we should not list in the events list ? I can remove the signature.

I do understand we need to listen to the event . I have an eg : https://gist.github.com/harikt/110dbf61cf089d3585f5 created with the help from irc by @josegonzalez .